jsapp/scss/stylesheets/partials/_base.scss

```diff
@@ -1,3 +1,4 @@
+@use 'scss/_colors';
```

Comment:
Please use `@use '~kobo-common/src/styles/colors';` instead

Reply:
Merging beta into the branch fixed it :)